habitat-lab/habitat/config/habitat/task/rearrange/cat_pick.yaml

```diff
@@ -41,6 +41,7 @@ end_on_success: True
 actions:
   arm_action:
```

Comment:
The arm controller of Stretch needs to be arm_controller: "ArmRelPosKinematicReducedActionStretch". You could do something like this:
  task:
    actions:
      arm_action:
        type: "ArmAction"
        arm_controller: "ArmRelPosKinematicReducedActionStretch"
        grip_controller: "MagicGraspAction"
        arm_joint_mask: [1,0,0,0,1,1,1,1,1,1]
        arm_joint_dimensionality: 10
        grasp_thresh_dist: 0.15
        disable_grip: False
        delta_pos_limit: 0.0125
        ee_ctrl_lim: 0.015
If you somehow can run this using the setting you used in interactive_play.py, then it should be fine.

Reply:
Thanks for providing these params. I switched to using kinematic arm controller and I will share the results I get.